creator-node/src/snapbackSM/secondarySyncHealthTracker.js

```diff
@@ -86,8 +93,8 @@ const Utils = {
 
   _parseRedisKeyIntoComponents(key) {
     const components = key.split(':::')
-    const [, secondary, wallet, syncType, outcome, date] = components
```

Comment:
this is where the bug was -- because the key is put into redis in the reverse order